.github/pull_request_template.md

```diff
@@ -2,7 +2,6 @@
 
 - [ ] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
 - [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
-- [ ] Did you write any [tests](https://github.com/open-ce/open-ce/blob/main/tests/) to validate this change?  
 
 ## Description
 
```

Comment:
We could probably remove thi sfile all together. The template [here](https://github.com/open-ce/.github/blob/main/pull_request_template.md) will be used if this file isn't here, and it's probably better to have that coming from one place.

Reply:
Ah, is that the same for the open-ce-builder one as well?

Reply:
Yeah, any repo in the org will use what's in the `.github` repo if they don't have one defined. Ecen all the feedstocks.

Reply:
I think maybe the one in open-ce just stayed there because it was there before the one in `.github`